python/ModelTools.py

```diff
@@ -86,6 +86,9 @@ def __init__(self,datacard,options):
         ModelBuilderBase.__init__(self,options) 
         self.DC = datacard
         self.doModelBOnly = True
+        self.selfNormBins = []
```

Comment:
selfNormBins keeps track of TH1-based channels. This is laying the groundwork for making combinations with a mixture of RooRealSumPdf and RooAddPdf channels.